study.md

```diff
@@ -25,7 +25,7 @@ In your own words, define what a SPA is. In your answer, be sure to cite any
 relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+Single Page Applications are web applications that can be modified/changed as the user interacts with it.
```

Comment:
Yes and the key element is essentially making one request to the API to get all the data back that comprises the app. Then the front end can decide what view states to show the user depending on where they are in the app without making extra requests to the server.